pkg/sdk/application.go

```diff
@@ -0,0 +1,37 @@
+package sdk
+
+import (
+	"context"
+
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+)
+
+// ApplicationSuppliers is the structure that represents a session's end block height
+// and its matching suppliers.
+type ApplicationSuppliers struct {
```

Comment:
I can't see how this is relevant nor how it's used.

Should we just remove this struct?